.github/workflows/release.yml

```
@@ -0,0 +1,144 @@
+          submodules: recursive
+
+      - name: Generate Changelog
+        id: changelog
+        uses: orhun/git-cliff-action@v2
+        with:
+          config: cliff.toml
+          args: --latest --strip header
+        continue-on-error: true  # 即使生成changelog失败也继续执行
+
+      - name: Download All Artifacts
+        uses: actions/download-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)